gtda/images/preprocessing.py

```diff
@@ -201,3 +201,115 @@ def transform(self, X, y=None):
         Xt = np.concatenate(Xt)
 
         return Xt
+
+
+@adapt_fit_transform_docs
+class ImageToPointCloud(BaseEstimator, TransformerMixin):
+    """Transformer returning a collection of point clouds corresponding to the
+    coordinates of the activated pixels of the 2D or 3D binary images of an
+    input collection.
+
+    Deactivated pixels are given infinite coordinates in that space.
+
+    Notes
```

Comment:
Elsewhere, I think we placed "Notes" after "Attributes" and before "References" (but either before or after "See also", somewhat inconsistently).